### ISSUE TITLE
Pseudo label training

### ISSUE THREAD

**Issue by dimimal** [open-mmlab/mmdetection#5116]
Is it possible to use this library and train with pseudo labels? 
Instead of training the model on robust labels, train the model with weak labels instead.
Thanks

**Comment by ZwwWayne** [comment]
Yes it is possible! You can use mmdet to train a model and generate pseudo labels, converting the results into COCO format, and load those results for training models. You can also generate pseudo labels online if you have enough GPU memory.

**Comment by kentaroy47** [comment]
@ZwwWayne 
> You can also generate pseudo labels online if you have enough GPU memory.
Interesting! Is there a document on doing this? Thanks!

**Comment by dimimal** [comment]
@ZwwWayne  If you have any reference of how to do this in mmdet I will be very pleased! Thanks!
